worldedit-core/src/main/java/com/fastasyncworldedit/core/queue/implementation/blocks/CharBlocks.java

```diff
@@ -20,6 +20,7 @@ public char[] get(CharBlocks blocks, int layer) {
             char[] arr = blocks.blocks[layer];
             if (arr == null) {
                 // Chunk probably trimmed mid-operations, but do nothing about it to avoid other issues
+                blocks.sections[layer] = EMPTY;
```

Comment:
I got a `StackOverflowError` without this change due to the section being full but the array being null

Reply:
It's probably best to use the getSkipFull method - we probably don't want to touch the sections array here? I make the change in #2912 as it is anyway

Reply:
Remioved the change